### ISSUE TITLE
Translation not working in 2024.6

### ISSUE THREAD

**Issue by KayMaik** [PiotrMachowski/Home-Assistant-custom-components-Custom-Templates#26]
Since the last Update of HA 2024.6 the translations did not working anymore.
Here is the logged protocoll.
```yaml
Traceback (most recent call last):
  File "/usr/src/homeassistant/homeassistant/helpers/template.py", line 603, in async_render
    render_result = _render_with_context(self.template, compiled, **kwargs)
                    ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/src/homeassistant/homeassistant/helpers/template.py", line 2616, in _render_with_context
    return template.render(**kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.12/site-packages/jinja2/environment.py", line 1304, in render
    self.environment.handle_exception()
  File "/usr/local/lib/python3.12/site-packages/jinja2/environment.py", line 939, in handle_exception
    raise rewrite_traceback_stack(source=source)
  File "<template>", line 2, in top-level template code
  File "/usr/local/lib/python3.12/site-packages/jinja2/sandbox.py", line 394, in call
    return __context.call(__obj, *args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/config/custom_components/custom_templates/__init__.py", line 89, in __call__
    translations = get_cached_translations(self._hass, language, "entity_component")
                   ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/config/custom_components/custom_templates/__init__.py", line 262, in get_cached_translations
    return cache.ct_patched_get_cached(language, category, components)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/config/custom_components/custom_templates/__init__.py", line 220, in get_cached
    category_cache = self.cache.get(language, {}).get(category, {})
                     ^^^^^^^^^^
AttributeError: '_TranslationCache' object has no attribute 'cache'

The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "/usr/src/homeassistant/homeassistant/helpers/template.py", line 715, in async_render_to_info
    render_info._result = self.async_render(  # noqa: SLF001
                          ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/src/homeassistant/homeassistant/helpers/template.py", line 605, in async_render
    raise TemplateError(err) from err
homeassistant.exceptions.TemplateError: AttributeError: '_TranslationCache' object has no attribute 'cache'
```

**Comment by costa-alex** [comment]
Same error here.

**Comment by PiotrMachowski** [comment]
Fixed in [v1.4.0](https://github.com/PiotrMachowski/Home-Assistant-custom-components-Custom-Templates/releases/tag/v1.4.0)
